app/views/organisations/show.html.erb

```diff
@@ -25,9 +24,8 @@
       <% end %>
       <% if @pending_org_admin %>
         <p class="org-info-msg-1" ><strong>Your request for admin status is pending.</strong></p>
-      <% end%> 
+      <% end%>
     </div>
```

Comment:
After having a look to code on RubyMine and in inspect element I think they are lined up.